### PR TITLE
[BIOMAGE-2211] - Update repo to use FluxV2

### PR DESCRIPTION
**Pull request by aerlaut**
# Description
We have to update the Helm charts because Flux v2 removes the use of HelmOperator and uses HelmController instead.

# Details
#### URL to issue
https://biomage.atlassian.net/browse/BIOMAGE-2211

#### Link to staging deployment URL (or set N/A)
N/A

#### Links to any PRs or resources related to this PR
N/A

#### Integration test branch
master

# Merge checklist
Your changes will be ready for merging after all of the steps below have been completed.

### Code updates
Have best practices and ongoing refactors being observed in this PR
- [ ] Migrated any selector / reducer used to the new format.

### Manual/unit testing
- [x] Tested changes using InfraMock locally **or** no tests required for change, e.g. Kubernetes chart updates.
- [ ] Validated that current unit tests for code work as expected and are sufficient for code coverage **or** no unit tests required for change, e.g. documentation update.
- [ ] Unit tests written **or** no unit tests required for change, e.g. documentation update.

### Integration testing
**You must check the box below** to run integration tests on the latest commit on your PR branch.
Integration tests have to pass before the PR can be merged. Without checking the box, your PR
**will not pass** the required status checks for merging.

- [ ] Started end-to-end tests on the latest commit.

### Documentation updates
- [ ] Relevant Github READMEs updated **or** no GitHub README updates required.
- [ ] Relevant Wiki pages created/updated **or** no Wiki updates required.

### Optional
- [ ] Staging environment is unstaged before merging.
- [ ] Photo of a cute animal attached to this PR.

### CODE REVIEW

File: chart-infra/templates/pod-template.tpl [hunk 1]
```diff
@@ -7,7 +7,7 @@
     spec:
       containers:
       - name: "{{ .Release.Name }}-r"
-        image: "{{ .Values.r.image }}"
+        image: "{{ .Values.r.image.registry }}/{{ .Values.r.image.repository }}:{{ .Values.r.image.tag }}"
```
Comment (ivababukova) [inline chart-infra/templates/pod-template.tpl:10]: what does this change do?

Comment (ivababukova) [inline chart-infra/templates/pod-template.tpl:10]: can you show me an example of an image? Just to see if this matches

Comment (aerlaut) [reply chart-infra/templates/pod-template.tpl:10]: image

Example of an image

Comment (aerlaut) [reply chart-infra/templates/pod-template.tpl:10]: This is a chart, so it'll be filled the values in HelmRelease, for example:

https://github.com/biomage-org/flux-v2-migration/blob/master/staging-candidates/worker/refs-pull-41-merge.yaml

image

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -378,7 +449,7 @@ jobs:
           API_TOKEN_GITHUB: ${{ secrets.API_TOKEN_GITHUB }}
         with:
           source_file: ${{ needs.build-docker.outputs.ref-id }}.yaml
-          destination_repo: '${{ github.repository_owner }}/releases'
+          destination_repo: '${{ github.repository_owner }}/flux-v2-migration'
```
Comment (ivababukova) [inline .github/workflows/ci.yaml:452]: change this to releases before merging

**Review by ivababukova — APPROVED** [review]

**Pull request merged**